src/Language/SourceLocation.php

```diff
@@ -57,11 +57,12 @@ public static function fromSource(Source $source, int $position): self
         $line    = 1;
         $column  = $position + 1;
         $matches = [];
-        preg_match_all("/\r\n|[\n\r]/", mb_substr($source->getBody(), 0, $position), $matches, PREG_OFFSET_CAPTURE);
+
+        \preg_match_all("/\r\n|[\n\r]/", \substr($source->getBody(), 0, $position), $matches, PREG_OFFSET_CAPTURE);
 
         foreach ($matches[0] as $index => $match) {
             $line   += 1;
-            $column = $position + 1 - ($match[1] + mb_strlen($match[0]));
+            $column = $position + 1 - ($match[1] + \strlen($match[0]));
```

Comment:
Why can we use strlen here but have to use mb_strlen otherwise? Can't we use strlen() everywhere considering that the graphql spec doesn't allow multibyte characters anyways afaik?

Reply:
@fubhy could you make a separate issue for this?